src/patterns/dosdp-patterns/cyclingCellStates.yaml

```diff
@@ -0,0 +1,64 @@
+pattern_name: cyclingCellStates
+pattern_iri: http://purl.obolibrary.org/obo/cl/cyclingCellStates.yaml
+description: "This pattern is for cells that are cycling/proliferating."
+
+contributors:
+  - https://orcid.org/0000-0001-6677-8489  # Aleix Puig
+
+classes:
+  cell: "CL:0000000"
+  M phase: "GO:0000279"
```

Comment:
I am afraid 'M phase' is too narrow to define cycling/proliferating cells.
Many markers that are widely used to detect cycling/proliferating cells in 'S phase' (*e.g.* [BrdU](https://en.wikipedia.org/wiki/Bromodeoxyuridine) in flow cytometry, histology; anti-[PCNA](https://en.wikipedia.org/wiki/Proliferating_cell_nuclear_antigen); anti-[MCM2](https://en.wikipedia.org/wiki/MCM2), *etc.*). I am not sure if any cell-cycle phase would work for this pattern, because they would be too narrow.
Please, consider
GO:0022402 cell cycle process: [The cellular process that ensures successive accurate and complete genome replication and chromosome segregation.](https://www.ebi.ac.uk/ols4/ontologies/go/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FGO_0022402?lang=en).

Reply:
Thank you for your input @rays22. It was [discussed in the issue](https://github.com/obophenotype/cell-ontology/issues/2354#issuecomment-2075513180) and in a call about which GO term could we use.
That is a good point and I agree that cycling cells can be in 'S phase', but the problem of 'cell cycle process' is that it also involve the G0 transition.
What about [mitotic cell cycle process](http://purl.obolibrary.org/obo/GO_1903047)? It also involves DNA replication: 
> A process that is part of the mitotic cell cycle.

@rays22 @dosumis

Reply:
> cycling cells can be in 'S phase', but the problem of 'cell cycle process' is that it also involve the G0 transition.

Cycling/proliferating cells go through **all** phases of the process, so I do not see any contradiction. These subdivisons of the cell cycle are very useful didactically, but they are also somewhat arbitrary and there are so many cell types in various species where some phases are skipped or irregular.

The narrower 'GO:1903047 mitotic cell cycle process' would work for most cell types, except for cells cycling/proliferating meiotically. If you make it clear in the text templates and labels what the pattern applies to (e.g. `mitotically cycling %`), than I am fine with using `mitotic cell cycle process`.

Reply:
@rays22 @aleixpuigb I think the intended definition of proliferating or cycling cells was the gene expression profile signature derived from single cell RNA sequencing analysis rather than measuring the DNA replication status directly, which is another way to assay it.  
Riba, A., Oravecz, A., Durik, M. et al. Cell cycle gene regulation dynamics revealed by RNA velocity and deep-learning. Nat Commun 13, 2865 (2022). https://doi.org/10.1038/s41467-022-30545-8

Reply:
> @rays22 @aleixpuigb I think the intended definition of proliferating or cycling cells was the gene expression profile signature derived from single cell RNA sequencing analysis rather than measuring the DNA replication status directly, which is another way to assay it. Riba, A., Oravecz, A., Durik, M. et al. Cell cycle gene regulation dynamics revealed by RNA velocity and deep-learning. Nat Commun 13, 2865 (2022). https://doi.org/10.1038/s41467-022-30545-8

@emquardokus , Just to clarify, the discussion here is about which specific GO process term to use, not about markers of M phase, S phase. It is about Gene Ontology terms that would fit the best.
The definition of cycling/proliferating cells involves one or more processes that are not specific to single cell RNA sequencing, or any particular assay.
The Gene Ontology already includes several biological process terms related to cycling/proliferation and they all look applicable to single cell expression pattern based categories as well. I can confirm that they also apply to the cell types in the paper you reference.
Using GO component terms in the axioms also ensures interoperability of expression and other types of biological data, such as linking gene expression to genetic perturbations.